staging/src/k8s.io/api/flowcontrol/v1alpha1/types.go

```diff
@@ -0,0 +1,297 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// These are valid flow-dingtinguisher methods.
+const (
+	// FlowDistinguisherMethodByUser specifies that the flow distinguisher is the username works in the request context
+	// so that the requests from the same user will enqueued into the same set of queues for processing.
+	FlowDistinguisherMethodByUserType FlowDistinguisherMethodType = "ByUser"
+
+	// FlowDistinguisherMethodByNamespace works by computing flow distinguisher with the user requested namespace. Any
```

Comment:
object's requested namespace?

Reply:
Yes, I think this could be improved.

Reply:
fixed in #81788

Reply:
Wouldn't "requested object's namespace" be better?
I think edits to the first line got tangled up in the third line; the latest rev (#81834) looks a little garbled here.